src/api/providers/fetchers/aistupidlevel.ts

```diff
@@ -0,0 +1,151 @@
+import axios from "axios"
+import { z } from "zod"
+
+import type { ModelInfo } from "@roo-code/types"
+
+import { parseApiPrice } from "../../../shared/cost"
+
+/**
+ * AIStupidLevelModel
+ */
+
+const aiStupidLevelModelSchema = z.object({
+	id: z.string(),
+	name: z.string().optional(),
+	description: z.string().optional(),
+	context_window: z.number().optional(),
+	max_tokens: z.number().optional(),
+	pricing: z
+		.object({
+			input: z.string().optional(),
+			output: z.string().optional(),
+		})
+		.optional(),
+})
+
+export type AIStupidLevelModel = z.infer<typeof aiStupidLevelModelSchema>
+
+/**
+ * AIStupidLevelModelsResponse
+ */
+
+const aiStupidLevelModelsResponseSchema = z.object({
+	data: z.array(aiStupidLevelModelSchema),
+})
+
+type AIStupidLevelModelsResponse = z.infer<typeof aiStupidLevelModelsResponseSchema>
+
+/**
+ * getAIStupidLevelModels
+ */
+
+export async function getAIStupidLevelModels(apiKey?: string): Promise<Record<string, ModelInfo>> {
```

Comment:
Given that this is required to get anything useful, this should be a required parameter